tests/e2e/config/wdio.android.rdc.conf.js

```diff
@@ -0,0 +1,32 @@
+const { config } = require('./wdio.rdc.shared');
+
+// ============
+// Capabilities
+// ============
+// For all capabilities please check
+// http://appium.io/docs/en/writing-running-appium/caps/#general-capabilities
+config.capabilities = [
+  {
+    deviceName: 'Google Pixel',
+    automationName: 'UiAutomator2',
+    // The reference to the app
+    testobject_app_id: '1',
+    // The api key that has a reference to the app-project in the TO cloud
+    testobject_api_key: process.env.SAUCE_RDC_EU_ACCESS_KEY_ANDROID,
+    // The name of the test for in the cloud
+    testobject_test_name: 'sample-app-mobile',
+    // Some default settings
+    // You can find more info in the TO Appium Basic Setup section
+    platformName: 'Android',
```

Comment:
No platform version here - do we want to always be testing on latest? (which for this device is Oreo)

Reply:
For now we can just pick a Pixel device in the cloud with dynamic allocation.